exporters/committime/collector_image.py

```diff
@@ -120,10 +120,18 @@ def commit_metric_from_image(self, app: str, image, errors: list) -> CommitMetri
     def _set_commit_timestamp(self, metric, errors) -> CommitMetric:
         # Only convert when commit_time is in metric, previously should be
         # found from the Label with fallback to annotation
-        if metric.commit_time:
-            metric.commit_timestamp = pelorus.convert_date_time_to_timestamp(
-                metric.commit_time, self._timedate_format
+        try:
+            if metric.commit_time:
+                metric.commit_timestamp = pelorus.convert_date_time_to_timestamp(
+                    metric.commit_time, self._timedate_format
+                )
+        except ValueError:
+            errors.append(
+                "COMMIT_DATE_FORMAT does not match annotation commit.date on the image\n"
             )
+            errors.append("Removing the offending timestamp")
```

Comment:
This is supposed to be a list of separate errors, not a logger. This should be the same call.